styles/style.css

```diff
@@ -0,0 +1,246 @@
+* {
+    margin:0;
+    padding: 0;
+}
+
+.centered {
+    display: flex;
+    justify-content: center;
+}
+
+body {
+    font-family: sans-serif;
+    font-size: 4em;
+}
+
+.parallax_wrapper {
+    height:100vh;
+    overflow-x: hidden;
+    overflow-y: auto;
+    perspective: 300px;
+}
+
+.intro_outro_screen {
+    height: 100vh;
+    align-items: center;
+
+}
+
+.parallax_group {
+    position: relative;
+    height: 100vh;
+    transform-style: preserve-3d;
+}
+
+.parallax_layer {
+    position: absolute;
+    inset: 0;
+}
+
+.top_layer {
+     /* scale calculation: 1 + ((Z transform value * -1) / perspective) */
+    transform: translateZ(110px) scale(0.63);
+}
+
+.mid_layer {
+    transform: translateZ(0);
+}
+
+.base_layer {
+    /* scale calculation: 1 + ((Z transform value * -1) / perspective) */
+    transform: translateZ(-300px) scale(2); 
+}
+
+
+#intro {
+    z-index: 10;
+    background-color: hotpink;
+}
+
+
+/* #group_one {
+    z-index: 5;
+} */
+.group_one_z_index {
+    z-index: 5;
+}
+#group-one > .base_layer {
+    background-color: lightskyblue;
+    
+}
+#group-one > .mid_layer {
+    /* background-color: cyan; opacity: 0.5; */
+}
+#group-one > .top_layer {
+    /* background-color: magenta; opacity: 0.5; */
+}
+
+
+/* #group_two {
+    z-index: 0;
+} */
+/* .group_two_z_index {
+    z-index: 0;
+} */
+#group-two > .base_layer {
+
+    background-color: rgb(114, 14, 202); 
+}
+#group-two > .mid_layer {
+    /* background-color: magenta; opacity: 0.5; */
+}
+#group-two > .top_layer {
+    /* background-color: yellow; opacity: 0.5; */
+}
+
+
+/* #group_three {
+    z-index: -5;
+} */
+/* .group_three_z_index {
+    z-index: -5;
+} */
+#group-three > .top_layer {
+    /* background-color: yellow; opacity: 0.5; */
+}
+#group-three > .base_layer {
+    background-color: lightgreen;
+}
+#group-three > .mid_layer {
+    /* background-color: magenta; opacity: 0.5; */
+}
+
+#outro {
+    z-index: 10;
+    background-color: rgb(255, 255, 147);
+}
+
+.container {
+    margin: 30px;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    flex-direction: row;
+}
+
+.group_title {
+    box-sizing: border-box;
+    padding-top: 4vh;
+    color: black;
+}
+.mid_layer_blurb {
+    font-size: 20px;
+    color: black;
+    width: calc(100% - 330px);
```

Comment:
You had mentioned wanting this text to wrap for more narrow screens. One way to accomplish that is using `@media` queries to apply different rules at different screen sizes. Another thing could be to try adding a `min-width` to this element, since we saw that when it was too wide for the container, that forced a wrap.